roles/pulp-resource-manager/defaults/main.yml

```diff
@@ -1,3 +1,5 @@
 ---
 pulp_resource_manager_state: started
 pulp_resource_manager_enabled: true
+pulp_config_dir: "/etc/pulp"
+pulp_settings_file: "{{ pulp_config_dir }}/settings.py"
```

Comment:
The role's README.md needs an update about the loose coupling. See pulp-content/README.md's "Shared variables"